packages/flutter/lib/src/widgets/scrollbar.dart

```diff
@@ -1634,16 +1634,16 @@ class RawScrollbarState<T extends RawScrollbar> extends State<T> with TickerProv
     late double primaryDelta;
     switch (position.axisDirection) {
       case AxisDirection.up:
-        primaryDelta = _dragScrollbarAxisOffset!.dy - updatedOffset.dy;
+        primaryDelta = clampDouble(_dragScrollbarAxisOffset!.dy, 0, position.viewportDimension) - updatedOffset.dy;
```

Comment:
I don't think `clamp`ing is the correct or expected behavior, especially in MacOS (haven't tested in Windows or Linux). As per the video below, which is Wikipedia open in Safari on MacOS Monoterey 12.4, looks like the last position where the mouse was "clicking" just before the scrollbar was about to overflow is saved and scrolling can resume only when the mouse crosses the last position.


https://user-images.githubusercontent.com/44755140/180601757-2aab6011-7a26-48ee-8038-fb4e2d2eeffc.mov

Reply:
Good catch, I will look into it.

Reply:
macOS with trackpad behaves differently

Reply:
looks same to me, mouse or touchpad. do you mind sharing a video?

Reply:
My colleague @huycozy tested this on all three platforms (Windows, macOS, and Linux) and concluded it's the same behavior on the other two platforms as @werainkhatri pointed out, thanks! 


Based on these new findings, I will figure out the new scrolling scrollbar behavior.